src/dom.js

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable arrow-body-style */
+import getWeatherInfo from './weather';
+
+function showMessage(msg) {
+  // eslint-disable-next-line no-alert
+  window.alert(msg);
+}
+
+function showLoading() {
+  const picture = document.querySelector('#loading-icon');
+  picture.classList.remove('hidden');
+}
+
+async function timeout(ms) {
+  showLoading();
+  await new Promise((resolve) => {
+    return setTimeout(resolve, ms);
+  });
+}
+
+function hiddenLoading() {
+  const picture = document.querySelector('#loading-icon');
+  picture.classList.add('hidden');
+}
+function clearWeatherInfo() {
+  const weatherSection = document.querySelector('#weather-section');
+  weatherSection.innerHTML = '';
+}
+function showWeather(w) {
+  hiddenLoading();
+  const wDesc = w.weather[0].description;
+  const wIcon = w.weather[0].icon;
+  const wImg = `http://openweathermap.org/img/wn/${wIcon}@2x.png`;
+  const humidity = `${w.main.humidity} %`;
+  const pressure = `${w.main.pressure} hPa`;
+  const tempC = Math.round(parseFloat(w.main.temp) - 273.15);
+  const tempMaxC = Math.round(parseFloat(w.main.temp_max) - 273.15);
+  const tempMinC = Math.round(parseFloat(w.main.temp_min) - 273.15);
+
+  // const celcius = Math.round(parseFloat(w.main.temp)-273.15);
+  // const fahrenheit = Math.round(((parseFloat(w.main.temp)-273.15)*1.8)+32);

```

Comment:
Remove these comments

Reply:
Thank you! Azdren. I did all the changes that you suggested. Now the app looks much better.
Let me know if something could be improved.